runtimes/standalone/src/lib.rs

```diff
@@ -344,25 +344,21 @@ impl frame_support::traits::OnRuntimeUpgrade for DelegationStructRuntimeUpgrade
 impl did::Config for Runtime {
 	/// The ubiquitous event type.
 	type Event = Event;
-	/// Type for the public signing key in DIDs
-	type PublicSigningKey = Hash;
-	/// Type for the public boxing key in DIDs
-	type PublicBoxKey = Hash;
 	type WeightInfo = ();
 }
 
-pub struct DidStructRuntimeUpgrade;
-impl did::migration::V23ToV24 for DidStructRuntimeUpgrade {
-	type PublicSigningKey = Hash;
-	type PublicBoxKey = Hash;
-	type AccountId = AccountId;
-	type Module = Attestation;
-}
-impl frame_support::traits::OnRuntimeUpgrade for DidStructRuntimeUpgrade {
-	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		did::migration::apply::<Self>()
-	}
-}
+// pub struct DidStructRuntimeUpgrade;
+// impl did::migration::V23ToV24 for DidStructRuntimeUpgrade {
+// 	type PublicSigningKey = Hash;
+// 	type PublicBoxKey = Hash;
+// 	type AccountId = AccountId;
+// 	type Module = Attestation;
+// }
+// impl frame_support::traits::OnRuntimeUpgrade for DidStructRuntimeUpgrade {
+// 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+// 		did::migration::apply::<Self>()
+// 	}
+// }
```

Comment:
better to delete that right away

Reply:
I haven't really looked at migrations yet, and perhaps we would do that together, that is why I simply commented it out.

Reply:
It's still here. This can get deleted. :)